nautobot_golden_config/templates/nautobot_golden_config/content_template.html

```diff
@@ -81,6 +81,12 @@
                             <a href="{% url 'plugins:nautobot_golden_config:configcompliance_details' pk=device.pk config_type='intended' %}"><i class="mdi mdi-text-box-check-outline"></i></a>
                         </td>
                     </tr>
+                    <tr>
+                        <td>Configuration to Push</td>
+                        <td>
+                            <a href="{% url 'plugins:nautobot_golden_config:configcompliance_details' pk=device.pk config_type='config_to_push' %}"><i class="mdi mdi-text-box-check-outline"></i></a>
```

Comment:
can you provide a screenshot?

Reply:
Having seen the screenshot, can we try and find another icon to differentiate? Additionally, should this should be in the "action buttons" on the Golden Config Home page.

Reply:
done